tests/test_library/kickstart-test/scenarios/test_plans/daily-rhel8.plan.yaml

```diff
@@ -0,0 +1,12 @@
+name: daily rhel8
+description: Daily run on current RHEL 8 development compose
+point_person: rvykydal@redhat.com
+artifact_type: github.scheduled.daily.kstest
+execute_on:
+  - filter: event.scenario.testplan == "rhel8"
```

Comment:
I know this is just test, but event should not specify testplan it runs. It should be the other way around. So the atrtifact type should be github.scheduled.daily.kstest.rhel8 or .daily-iso and the testplan can be selected just by that.